surveyor/surveyor_test.go

```diff
@@ -656,7 +656,7 @@ func TestSurveyor_ObservationsWatcher(t *testing.T) {
 	if err := os.Mkdir(dirName, 0700); err != nil {
 		t.Fatalf("Error creating observations dir: %s", err)
 	}
-	defer os.RemoveAll(dirName)
+	// defer os.RemoveAll(dirName)
```

Comment:
Did you mean to leave this commented out

Reply:
thanks for catching that!